server/services/auth.js

```diff
@@ -0,0 +1,39 @@
+import jwt from 'jsonwebtoken';
+
+import config from '../../config/env';
+import User from '../models/user';
+import userService from '../services/user';
+
+function loginUser(email) {
+  return User.getByEmail(email)
+    .then((user) => {
+      const token = jwt.sign({ email }, config.auth.jwt_secret);
+      const returnObj = {
+        user: user,
+        token: token
+      };
+      return returnObj;
+    });
+}
+
+function signupUser(body) {
+  const userObj = {
+    email: body.email,
```

Comment:
It's also a password. The password isn't currently used bc this is a dummy authorization workflow, though.